### PR TITLE
Rebase on upstream

### PR DESCRIPTION
**Pull request by scjody**
Closes https://github.com/plotly/streambed/issues/12867

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +86,24 @@ make test-heroku-16
...
+    git rebase heroku/master master
+    git push -f origin master
+    ```
+- Create a tag. This tag should correspond with the current herokuish release to indicate what release we're building on top of.
+    ```shell
+    # if latest gliderlabs/herokuish is 0.4.5
+    git tag 0.4.5-1
+    git push --tags
+    ```
+- CI will build a release and push to quay.io
```
Comment (josegonzalez) [inline README.md:107]: This isn't quite true anymore. The release process for the herokuish image is now in plotly/herokuish.

Comment (scjody) [reply README.md:107]: Yes, this should have been updated when the CircleCI hooks were removed, but I'll address that in a separate PR.

**Review by josegonzalez — APPROVED** [review]
💃 this seems reasonable to me :)

**Comment by scjody** [comment]
* [x] Tested by deploying a new app

**Pull request merged**